test/integration/files.js

```diff
@@ -47,4 +47,17 @@ describe('files API', function () {
       updated.data.should.have.property('attributes')
     })
   })
+
+  describe('Create directory', function () {
+    it('Works', async function () {
+      if (config.isV2) {
```

Comment:
Can we move this check a bit upper? So, we can avoid to have `Create directory Works` in the logs when running the integration tests with cozy V2.